app/src/db/migrations/20210519170000_015-alter-submission-data-vw.js

```diff
@@ -0,0 +1,90 @@
+const previousMigration  = require('./20201019100738_007-public-submission-data-vw').up;
+
+exports.up = function (knex) {
+  return Promise.resolve()
+    // JOIN user table to submissions_vw on form_submission_status.assignedToUserId to get 'assigned to' user
+    .then(() => knex.schema.raw(`create or replace
+    view submissions_vw as
+    SELECT
+      s.id AS "submissionId",
+      s."confirmationId",
+      s.draft,
+      s.deleted,
+      s."createdBy",
+      s."createdAt",
+      f.id AS "formId",
+      f.name AS "formName",
+      fv.id AS "formVersionId",
+      fv."version",
+      st.id AS "formSubmissionStatusId",
+      st.code AS "formSubmissionStatusCode",
+      st."createdBy" AS "formSubmissionStatusCreatedBy",
+      st."createdAt" AS "formSubmissionStatusCreatedAt",
```

Comment:
let me know if this needs to be renamed eg: `createdAt`

Reply:
No that works thanks! Same format as the code and id columns. 👍